farscape.gemspec

```diff
@@ -27,6 +27,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'byebug'
   s.add_development_dependency 'redcarpet', '~> 3.3'
   s.add_development_dependency 'rspec', '~> 2.14'
+  s.add_development_dependency 'webmock'
```

Comment:
the version should be specified since v2 that was released a few weeks ago has a few api changes